pkg/cmd/cli/cmd/newapp.go

```diff
@@ -80,10 +84,12 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 	cmd.Flags().Var(&config.SourceRepositories, "code", "Source code to use to build this application.")
 	cmd.Flags().VarP(&config.ImageStreams, "image", "i", "Name of an OpenShift image stream to use in the app.")
 	cmd.Flags().Var(&config.DockerImages, "docker-image", "Name of a Docker image to include in the app.")
+	cmd.Flags().Var(&config.Templates, "stored-template", "Name of an OpenShift stored template to use in the app.")
+	cmd.Flags().Var(&config.TemplatesParameters, "stored-template-parameters", "Specify a list of key value pairs (eg. --stored-template-parameters=FOO=BAR,BAR=FOO) to set/override parameter values in the stored template.")
```

Comment:
These names are too long.  `--template` should be used (and the output version should be renamed `--output-template` for this command, or for now, just dropped).

`--stored-template-parameters` is about 20 characters too long.  I think this should be `--param` and `-p`